README.md

```diff
@@ -25,3 +25,55 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+
+## Features and Benefits
+
+## Annotations
+Display any HTML element as an annotation at a specific point of interest in the Angular Circular Gauge. You can also add multiple annotations in a gauge.
+
+## Appearance
+The appearance of each element in a Circular Gauge, such as axes, ranges, axes intervals, pointer positions, label positions, and tick positions, can be customized easily.
+
+## Legend
+A legend provides additional information that is helpful in identifying the ranges in a Circular Gauge. It can be docked at the left, right, top, and bottom positions around the plot area.
+
+## Tooltip
+The interactive Angular Circular Gauge control provides options to display details about a pointer value through a tooltip when hovering the mouse over the pointer.
+
+## Axes customization
+The gauge axes are circular scales where a set of values can be plotted based on any business logic. You can also easily customize the appearance of the axes.
+
+## Related links
+[Learn More about Angular Circular Gauge](https://www.syncfusion.com/angular-components/angular-circular-gauge?utm_source=github&utm_medium=listing&utm_campaign=angular-circular-gauge-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-circular-gauge-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/angular?utm_source=github&utm_medium=listing&utm_campaign=angular-circular-gauge-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/angular/documentation/circular-gauge/getting-started/?utm_source=github&utm_medium=listing&utm_campaign=angular-circular-gauge-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/ej2-circular-gauge-angular-getting-started.git?utm_source=github&utm_medium=listing&utm_campaign=angular-circular-gauge-github-samples)
```

Comment:
is this the correct link?
https://github.com/SyncfusionExamples/ej2-circular-gauge-angular-getting-started.git

Reply:
changed the demo link